frontend/src/metabase/browse/components/BrowseModels.styled.tsx

```diff
@@ -8,6 +8,7 @@ import Card from "metabase/components/Card";
 import { Ellipsified } from "metabase/core/components/Ellipsified";
 import Link from "metabase/core/components/Link";
 import { Group } from "metabase/ui";
+import IconButtonWrapper from "metabase/components/IconButtonWrapper";
```

Comment:
Do we want to be using the IconButtonWrapper? I see we're wrapping the Model icon inside a button, but I don't think it's clickable is it?

Reply:
Good catch, fixed